tests/fireloop/server/model-config.json

```diff
@@ -17,70 +17,70 @@
     ]
   },
   "User": {
-    "dataSource": "db"
+    "dataSource": "mongodb"
```

Comment:
Why is this change part of this pull request? it seems it squashed commits non related to this functionality.. This modifies the history